set.mm

```diff
@@ -481233,6 +481329,394 @@ such that every prime ideal contains a prime element (this is a
+      rspectopn.2 $e |- P = ( PrmIdeal ` R ) $.
+      rspectopn.3 $e |- J = ran ( i e. I |-> { j e. P | -. i C_ j } ) $.
+      $( The topology component of the spectrum of a ring.  (Contributed by
+         Thierry Arnoux, 4-Jun-2024.) $)
+      rspectopn $p |- ( R e. Ring -> J = ( TopOpen ` S ) ) $=
+        ( vy wcel ctopn cfv co cv eqid wceq cvv crg cidlsrg crest wss crab cmpt
+        vx wn crn cress crspec cprmidl rspecval oveq2i 3eqtr4g resstopn syl6eqr
+        fveq2d cts idlsrgtset cbs cpw wral wa clidl fvexi rabex simp2 idlsrgbas
+        a1i w3a adantr 3ad2ant1 eleqtrd rabssdv elpwd ralrimiva eqsstrrd topnid
+        rnmptss syl eqtrd oveq1d wrex cin mptex elrest mp2an rgenw ineq1 eqeq2d
+        wb rnex rexrnmptw ax-mp inrab2 prmidlssidl 3sstr4g sseqin2 sylib syl5eq
+        rabeqdv rexbidv syl5bb eleq2i elrnmpti bitri syl6bbr eqrdv 3eqtr2rd ) B
+        UAMZCNOZBUBOZNOZAUCPZDFDQZEQZUDUHZEFUEZUFZUIZAUCPZGXKXLXMAUJPZNOXOXKCYC
+        NXKBUKOXMBULOZUJPCYCBUMHAYDXMUJJUNUOURAYCXNXMYCRXNRUPUQXKYAXNAUCXKYAXMU
+        SOZXNBXMDEFYAUAXMRZIYARUTZXKYEXMVAOZVBZUDYEXNSXKYEYAYIYGXKXSYIMZDFVCYAY
+        IUDXKYJDFXKXPFMZVDZXSYHTXSTMZYLXREFFBVEIVFZVGZVJYLXREFYHYLXQFMZXRVKXQFY
+        HYLYPXRVHYLYPFYHSZXRXKYQYKBXMFUAYFIVIVLVMVNVOVPVQDFXSYIXTXTRZVTWAVRYHYE
+        XMYHRYERVSWAWBWCXKUGYBGXKUGQZYBMZYSXREAUEZSZDFWDZYSGMZYTYSLQZAWEZSZLYAW
+        DZXKUUCYATMATMYTUUHWLXTDFXSYNWFWMABULJVFZLYSAYATTWGWHUUHYSXSAWEZSZDFWDZ
+        XKUUCYMDFVCUUHUULWLYMDFYOWIUUGUUKDLFXSXTTYRUUEXSSUUFUUJYSUUEXSAWJWKWNWO
+        XKUUKUUBDFXKUUJUUAYSXKUUJXREFAWEZUEUUAXREFAWPXKXREUUMAXKAFUDUUMASXKYDBV
+        EOAFBWQJIWRAFWSWTXBXAWKXCXDXDUUDYSDFUUAUFZUIZMUUCGUUOYSKXEDFUUAYSUUNUUN
+        RXREAUUIVGXFXGXHXIXJ $.
+    $}
+  $}
+
+  ${
+    zarclsx.1 $e |- V = ( i e. ( LIdeal ` R )
+                                    |-> { j e. ( PrmIdeal ` R ) | i C_ j } ) $.
+
+    ${
+      $d .0. i j $.  $d P i $.  $d R i j $.
+      zarcls0.1 $e |- P = ( PrmIdeal ` R ) $.
+      zarcls0.2 $e |- .0. = ( 0g ` R ) $.
+      $( The closure of the identity ideal in the Zariski topology.
+         (Contributed by Thierry Arnoux, 16-Jun-2024.) $)
+      zarcls0 $p |- ( R e. Ring -> ( V ` { .0. } ) = P ) $=
+        ( crg wcel csn cv wss cprmidl cfv cvv wceq a1i wa crab cmpt wral simplr
+        clidl simpll prmidlidl sylancom lidl0cl syl2anc snssd eqsstrd ralrimiva
+        eqid rabid2 sylibr syl5req lidl0 fvexi fvmptd ) BJKZCFLZCMZDMZNZDBOPZUA
+        ZABUEPZEQECVHVGUBRVAGSVAVCVBRZTZAVFVGHVJVEDVFUCVFVGRVJVEDVFVJVDVFKZTZVC
+        VBVDVAVIVKUDVLFVDVLVAVDVHKZFVDKVAVIVKUFZVJVKVAVMVNVDBUGUHBVHVDFVHUNZIUI
+        UJUKULUMVEDVFUOUPUQBVHFVOIURAQKVAABOHUSSUT $.
+    $}
+
+    ${
+      $d B i j m $.  $d I i j m $.  $d R i j m $.  $d V m $.
+      zarcls1.1 $e |- B = ( Base ` R ) $.
+      $( The intersection of a family of closed sets is closed in the Zariski
+         topology.  (Contributed by Thierry Arnoux, 16-Jun-2024.) $)
+      zarcls1 $p |- ( ( R e. CRing /\ I e. ( LIdeal ` R ) )
```

Comment:
What is "family of closed sets "? Wahat is its "intersection"? I cannot see them in the theorem (and its hypotheses).

Reply:
Sorry, this is a copy/paste error. Changing to the correct comment:
> The unit ideal ` B ` is the only ideal which closure in the Zariski topology is the empty set.

Reply:
Fixed in last commit.